content/md/permissions-settings/access-rights-on-data-quality.md

```diff
@@ -0,0 +1,28 @@
+---
+id: access-rights-on-data-quality
+themes: permissions-settings
+title: Set rights on **data quality**
+popular: false
+ee-only: true
+related: understand-data-quality, improve-data-quality, manage-data-quality
+---
+
+# Overview
+You can easily restrict the activation and deactivation of data quality insights on attribute groups depending on the user groups your users are in.
+
+By default, all users with the right to edit an attribute group, have the right to activate and deactivate data quality insights.
+
+# Set rights to activate and deactivate data quality
+
+To set rights:
+1. Go to `System`  
+2. Click on `Roles`
+3. Select a `Role`
```

Comment:
You don't have to use the quotes ` ` when you're not mentioning the name of a button, or a menu, or a value/code/label.
```suggestion
3. Select a role
```